bats/tests/helpers/os.bash

```diff
@@ -84,16 +84,26 @@ sudo_needs_password() {
 
 supports_vz_emulation() {
```

Comment:
The asymmetry with `using_systemd` bothers me a little. This could be made to look much more similar with

```suggestion
supports_vz_emulation() {
    if ! is_macos; then
        return 1
    fi
    [[ -n ${_RD_SUPPORTS_VZ_EMULATION:-} ]] || load_var _RD_SUPPORTS_VZ_EMULATION || true
    if [[ -z ${_RD_SUPPORTS_VZ_EMULATION:-} ]]; then
        ...
    fi
    is_true "${_RD_SUPPORTS_VZ_EMULATION}"
}
```